minard/views.py

```diff
@@ -740,7 +766,7 @@ def check_rates_history():
     starting_run = request.args.get('starting_run',103214,type=int)
 
     data, stats = polling_history(crate, slot, channel, starting_run)
-    discriminator_threshold = get_discriminator_threshold(crate, slot, channel)
+    discriminator_threshold = get_discriminator_threshold(crate, slot)
     return render_template('check_rates_history.html', crate=crate, slot=slot, channel=channel, data=data, stats=stats, discriminator_threshold=discriminator_threshold)
```

Comment:
Is this not going to fail with the different return type? If so, maybe it's better to just create a new function get_discriminator_thresholds_for_slot or something.

Reply:
The function never used the channel. I was just removing unused code.